test/integration/plugin.cc

```diff
@@ -32,14 +36,62 @@
 #include "utils.hh"
 
 /////////////////////////////////////////////////
-TEST(Loader, LoadBadPlugins)
+bool BadPlugins(std::vector<std::string> &_paths)
 {
-  std::vector<std::string> libraries = {
+#ifdef IGNDummyPlugins_LIB
+  _paths = std::vector<std::string>{
     IGNBadPluginAPIVersionOld_LIB,
     IGNBadPluginAPIVersionNew_LIB,
     IGNBadPluginAlign_LIB,
     IGNBadPluginNoInfo_LIB,
     IGNBadPluginSize_LIB};
+  return true;
+#else
+  std::string dataDir;
+  if (ignition::common::env("TEST_SRCDIR", dataDir))
+  {
+    _paths = std::vector<std::string>{
+      ignition::common::joinPaths(dataDir,
+          "__main__/ign_plugin/test/IGNBadPluginAPIVersionOld.so"),
```

Comment:
These `__main__` paths may will look very confusing to non-Bazel developers.

Is there a way this can be put into a variable from the Bazel files, similar to how we pass values [from cmake to cpp](https://github.com/ignitionrobotics/ign-gazebo/blob/a5abacfa74db6e8db0218bd32c55980e15cf249b/test/test_config.hh.in#L23)?